tests/unit_tests/test_vectorstore.py

```diff
@@ -508,8 +990,6 @@ def test_pgvector_with_with_metadata_filters_6(
         {"$and": {}},
         {"$between": {}},
         {"$eq": {}},
-        {"$exists": {}},
```

Comment:
Why were these removed?

Reply:
It's fixed. Can you start the approval ?